package.xml

```diff
@@ -13,27 +13,32 @@
   <license>BSD-2</license>
   <url>https://github.com/loco-3d/linear_feedback_controller</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
 
-  <depend>roscpp</depend>
-  <depend>pinocchio</depend>
-  <depend>realtime_tools</depend>
-  <depend>example-robot-data</depend>
-  <depend>linear_feedback_controller_msgs</depend>
-  <depend>pal_base_ros_controller</depend>
+  <!-- Standard dependencies -->
+  <build_depend>eigen</build_depend>
+  <buildtool_export_depend>eigen3_cmake_module</buildtool_export_depend>
+  <build_export_depend>eigen</build_export_depend>
 
-  <!-- Need to depend directly on the package containing the base type of the controller for the plugin to work -->
+  <depend>rclcpp</depend>
+  <depend>rclcpp_lifecyle</depend>
   <depend>controller_interface</depend>
-
-  <!-- Create the plugin -->
+  <depend>hardware_interface</depend>
+  <depend>semantic_components</depend>
```

Comment:
```suggestion
```
There is no such package as `semantic_components`. They are part of `controller_interface`